index.html

```diff
@@ -4,10 +4,310 @@
+        </div>
+        </div>
+    
+    </section>
+          </div>
+          
+
+        <section class="about">
+        <div id="about">
+        <section id="about001">
+        <h2 class="my001">About<br>MySelf</h2>
+        <p class="intro">
+            Hello I am a software developer!
+             I can help you build a product
+              ,feature or website Look through
+               some of my work and experience!
+                if you like what you see and 
+                have a project you need coded,
+                 don't hesitate to contact me.
+        </p>
+        <div class="contact">
+            <h3>LET'S  CONNECT</h3>
+             </div>
+             <div class="icons">
+        <ul>
+          <li><img src="./image/image6.png" alt="twitter"></li>
+          <li><img src="./image/image7.png" alt="twitter"></li>
+          <li><img src="./image/image8.png" alt="twitter"></li>
+           <li><img src="./image/image9.png" alt="twitter"></li>
+          <li><img src="./image/imge5.png" alt="twitter"></li>
+        </ul>
+        </div> 
+          <button class="button" class="resume" type="button">Get My Resume</button>
+        </section>
+                <section id="about002">
+                <div class="end">
+                 <ul class="photo">
+                <li><span class="h6">Languages</span> <span class="icon"><img src="./image/imge8.png" alt="photo"></span></li>
+                </ul>
+                        <div class="fitems">
+                             <ul>
+                            <li><span><img src="./image/Ellipse1.png" alt="ellipse001"></span><span>Javascript</span></li>
+                           </ul>
+                            <ul>
+                            <li><span><img src="./image/Ellipse2.png" alt="ellipse002"></span> <span>CSS</span></li>
+                            </ul>
+                             <ul>
+                            <li><span><img src="./image/ellipse3.png" alt="ellipse003"></span> <span>HTML</span></li>
+                            </ul>
+                          </div>
+                          
+                          <div class="frames">
+            <ul class="photo1">
+                <li><span>Frameworks</span><span class="icon"><img src="./image/imge9.png" alt="photo"></span></li>
+            </ul>
+            <ul>
+                <li class="down"><img src="./image/imgg1.png" alt="line"></li>
+            </ul>
+            <ul class="photo2">
+                <li ><span>Skills</span><span class="icon"><img src="./image/imge9.png" alt="photo"></span> </li>
+            </ul>
+            <ul>
+                <li class="down"><img src="./image/imgg1.png" alt="line2"></li>
+            </ul>
+        </div>
+            </div>
+        </section>
+        </div>
+            <div class="form-contact">
+                 <center> <div class="info-form">
+                    <h2 class="h002">Contact me</h2>
```

Comment:
- [ ] Kindly consider centering your form title to be consistent with the design throughout your portfolio 👍🏽

### Your design
![image](https://user-images.githubusercontent.com/43172164/208076984-18b729cd-646f-4ad0-a328-b8793269fe8a.png)

### Figma template
![image](https://user-images.githubusercontent.com/43172164/208077126-dd4476bb-443b-45d2-ac0d-3b5a34e61fac.png)